.changes/nextrelease/webidentityexceptions.json

```diff
@@ -0,0 +1,7 @@
+[
+    {
+        "type": "bugfix",
+        "category": "CredentialProvider",
```

Comment:
The category targets the namespace (minus the `Aws`), so in this case it would be `Credentials`.